docs/modules/deployment-launcher/upload-assemblies.md

```diff
@@ -0,0 +1,65 @@
+<%( TOC )%>
+
+# Upload assemblies
+
+Open the Deployment Launcher window by selecting **SpatialOS** > **Deployment Launcher** from the Unity Editor menu.
+
+## UI walkthrough
+
+The section of the Deployment Launcher window that you use to upload assemblies can be found under the **Assembly Upload** label:
+
+<img src="{{assetRoot}}assets/modules/deployment-launcher/assembly-upload.png" style="margin: 0 auto; width: auto; display: block;" />
+
+<%(#Expandable title="Fields")%>
+
+| Field | Description |
+| --- | --- |
+| Assembly Name | This is an identifier for the assembly you will upload.<br/><br/>You can use this to reference an assembly when launching a deployment or to find the assembly in the [SpatialOS Console](https://docs.improbable.io/reference/latest/shared/glossary#console). |
+| Force Upload | Denotes whether to force upload this assembly.<br/><br/>If this is checked, an assembly that previously was uploaded with the same assembly name will be overwritten. |
+
+<%(/Expandable)%>
+
+<%(#Expandable title="Buttons")%>
+
+| Button name | Description |
+| --- | --- |
+| Generate | When pressed, this generates an **Assembly Name** for you based on your SpatialOS project name and a timestamp. |
+| Assign assembly name to deployments | When pressed, this copies the current value of **Assembly Name** to _all_ deployments you currently have [configured in the next section]({{urlRoot}}/modules/deployment-launcher/launch-deployments). |
```

Comment:
```suggestion
| Assign assembly name to deployments | When pressed, this copies the current value of **Assembly Name** to _all_ deployments you currently have configured as described in [the next section]({{urlRoot}}/modules/deployment-launcher/launch-deployments). |
```

or something similar, just find the usage of `currently` and `next` right next to each other slightly confusing